worldedit_commands/init.lua

```diff
@@ -387,6 +387,25 @@ minetest.register_chatcommand("/set", {
 	end, check_region),
 })
 
+minetest.register_chatcommand("/param2", {

```

Comment:
missing chat command documentation

Reply:
?????
/help /param2 works.
What are you talking about?

Reply:
`ChatCommands.md`

Reply:
Sounds like a redundancy to me. But if you insist …
Don't worry, I'll address all the complains later.